src/DependencyInjection/Configuration.php

```diff
@@ -214,4 +215,23 @@ private function createSmsConfiguration(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    private function createLocaleCookieConfiguration(ArrayNodeDefinition $root)
+    {
+        $root
+            ->children()
+                ->arrayNode('locale_cookie')
+                    ->info('Cookie settings for locale cookie')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('name')->end()
+                        ->scalarNode('domain')->end()
+                        ->integerNode('expire')->end()
+                        ->scalarNode('path')->end()
+                        ->booleanNode('secure')->end()
+                        ->booleanNode('httpOnly')->end()
```

Comment:
I don't know. There are no concrete reasons to configure secure to false. But I don't know all the use-cases of the stepup-bundle.
Neither for http_only to true. Or path to anything else than / or expire to anything but 0.

I can remove these configuration values, which means that if SURFnet does come up with a reason we'll have to modify the software.

Though I don't really see any security reasons for this write-only cookie for interop to withhold configuration.

@rjkip I don't have strong feelings either way. Remove it just for easier maintenance?